source/best-practices/index.rst

```diff
@@ -1,4 +1,11 @@
 前言
 ====
 
-这一部分介绍我们在日常科研工作中总结的一些经验，供读者参考。
+这一部分介绍我们在日常科研工作中整理和总结的一些经验，供读者参考。
+
+欢迎地震学新手在实践中尝试这些经验总结，并再次整理、总结和分享。
+最后形成实践、整理、总结、再实践、再整理、再总结……的良性循环。
+
+**实践是检验真理的唯一标准！**
+
+**整理和总结是提高科研生产率的有效催化剂~**
```

Comment:
Not sure what the "quote" syntax look like.
```suggestion
> 实践是检验真理的唯一标准！

> 整理和总结是提高科研生产率的有效催化剂！
```

Reply:
It seems that the quote does not work.

Reply:
Please try changing > to |

Reply:
use four space indetation and ** instead. see 5bda85f.

Reply:
> Please try changing > to |

｜does not work either.